### ISSUE TITLE
Hotkey is not working with router-links (at least not the easy way)

### ISSUE THREAD

**Issue by suterma** [Simolation/vue-hotkey#1]
I tried the "Button"-Example **with a router-link** instead:

                <Hotkey :keys="['ctrl', 's']" v-slot="{ clickRef }">
                    <router-link :ref="clickRef" class="navbar-item" to="/play"
                        >Play</router-link
                    >
                </Hotkey>

The expectation was, that the route would be pushed, as what happens when the user clicks on the generated link. Instead, I get the following error:

    vue-hotkey.es.js?97ff:211 Uncaught TypeError: _a2.click is not a function
        at Object.handler (vue-hotkey.es.js?97ff:211:1)
        at HTMLDocument.keydown (vue-hotkey.es.js?97ff:175:1)

**No routing action is taking place.**

I can do however:

                <Hotkey
                    :keys="['ctrl', 's']"
                    @hotkey="
                        () => {
                            this.$router.push('/play');
                        }
                    "
                >
                    <router-link
                        @click="toggleBurger"
                        class="navbar-item"
                        to="/play"
                        >Play</router-link
                    >
                </Hotkey>

Which works just fine, but takes the fun out of it, since now I could also just register my hotkey action directly on the DOM with a library like [vue-global-events](https://github.com/shentao/vue-global-events). 

Note: I already use this library for actions which are not directly tied to an input element. However I find your approach quite convincing for actions which ARE actually directly tied to an input element.

**Comment by Simolation** [comment]
Unfortunately, Vue router does not accept click events as it is not a native element but a custom component.
What you can do though is extend the router-link as described here: https://router.vuejs.org/guide/advanced/extending-router-link.html#extending-routerlink

```Vue
<Hotkey :keys="['ctrl', 's']" v-slot="{ clickRef }">
  <router-link
    to="/play"
    custom
    v-slot="{ href, navigate }"
  >
    <a
      :ref="clickRef"
      :href="href"
      class="navbar-item"
      @click="navigate"
    >
      Play
    </a>
  </router-link>
</Hotkey>
```

I have not tried this code, but is should work like that

**Comment by suterma** [comment]
The above code works (navigates to the view), but the new route is not properly visualized because the isActive property is missing. In the linked example this gets addressed and I will try it later. 

Maybe you want to specifically handle such situations more gracefully ( a warn message maybe) or handle this specifically  in your code or have this situation addressed in the docs.

**Comment by Simolation** [comment]
Actually, I just resolved the issue as it is, in fact, possible to access the HTML element of custom components 👍 
I will add one smaller other improvement and then release a new version

**Comment by Simolation** [comment]
It's released in v1.1.10

Thank you for pointing this issue out 😃

**Comment by suterma** [comment]
Thanks for providing this fix to an already useful package. It's working like I expected now.

I encourage you to still include an example in the docs, now sporting this feature.
